operator/pkg/util/merge_iop.go

```diff
@@ -161,6 +161,7 @@ type (
 	}
 	meshConfigExtensionProvider struct {
 		Name               string   `json:"string"`
+		EnvoyOtelAls       struct{} `json:"envoyOtelAls"`
```

Comment:
Fine to make forward progress but aren't there like 10 other ones that are not captured here?

Reply:
yes, I'm finding out a way to fix the others, need more time.

Reply:
@howardjohn  update the implement and add a test prevent miss implement once there's a new provider.